problem_2/main.ipynb

```diff
@@ -27,6 +27,7 @@
     "import pandas as pd\n",
```

Comment:
<p>Since we do standardize this - I do believe we can drop the <code>Returns_pct</code> and <code>Returns_val</code> columns. We do not really care for them right now, and if we do need them later on, we can just add them then. That said, If we decide to drop those columns; we should then add the <code>Normalizer</code> to the <code> StockCalculator.monthly_returns</code> function, and we can do it there. That way, we skip a lot of memory-consuming tasks like creating new dataframes all the time :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeanMarcusJensen/ACIT4610_Portifolio_KMHS/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>